app/lib/audit/catalog_to_archive.rb

```diff
@@ -0,0 +1,71 @@
+module Audit
+  # Catalog to cloud archive provider (what ZipEndpoint, ZippedMoabVersion etc represent) audit code
+  class CatalogToArchive
+    # delegate :bucket, :bucket_name, to: PreservationCatalog::S3
+
+    def self.logger
+      @logger ||= Logger.new(STDOUT)
```

Comment:
Do we need to reproduce this weird double-logging pattern?  If you note the places we actually schedule these jobs with `whenever`, we are basically routing the STDOUT portion to `/dev/null` anyway.

Reply:
discussed F2F, 👍 will just log to `c2a.log`